core/src/main/java/org/infinispan/xsite/BaseBackupReceiver.java

```diff
@@ -123,6 +125,20 @@ public Object visitReplaceCommand(InvocationContext ctx, ReplaceCommand command)
       }
 
       @Override
+      public Object visitComputeCommand(InvocationContext ctx, ComputeCommand command) throws Throwable {
+         log.tracef("Processing a remote compute %s", command);
```

Comment:
This trace message sticks out a bit. Thinking we should either remove or add logs to all the rest at some point.